remo/reports/tests/test_api.py

```diff
@@ -69,3 +73,104 @@ def test_get_remo_url(self):
         data = ActivitiesDetailedSerializer(
             report, context={'request': request}).data
         ok_(report.get_absolute_url() in data['remo_url'])
+
+
+class TestActivitiesKPIView(RemoTestCase):
```

Comment:
We can surely omit this. My point is not to test the DRF functionality, rather than test that your APIView only accepts get requests.